### DISCUSSION TITLE
Tokens are not accepted

### DISCUSSION THREAD

**Discussion by marineboy0122** [teslamate-org/teslamate#2692] [Q&A]
I entered a token value, but I get the error Error: Tokens are invalid.
![111](https://user-images.githubusercontent.com/83511237/172194210-28b36a5a-c18e-4961-b32a-cb256ccf5fc9.png)
![222](https://user-images.githubusercontent.com/83511237/172194218-6da98489-f96f-4b84-ad8b-0524ac51fd63.png)

docker teslamate log - [error] POST https://auth.tesla.com/oauth2/v3/token -> error: "non-existing domain" (8029.491 ms)

**Comment by buzzhd** [comment]
same here and even reset the password and still no go with the access and refresh token. Any fix?

**Reply by marineboy0122** [reply]
no,I haven't solved it yet.

**Comment by marineboy0122** [comment]
https://user-images.githubusercontent.com/83511237/175807931-f4df4800-e53d-4205-9f57-a94f0b49d21b.mp4
